tests/foreman/cli/test_org.py

```diff
@@ -1370,39 +1365,6 @@ def test_negative_update_1(self, test_data):
         self.assertGreater(len(result.stderr), 0,
                            "There should be error - hammer expects error")
 
-    @skip_if_bug_open('bugzilla', 1114136)
-    @data({'description': gen_string('alpha', 3000)},
-          {'description': gen_string('numeric', 3000)},
-          {'description': gen_string('alphanumeric', 3000)},
-          {'description': gen_string('utf8', 3000)},
-          {'description': gen_string('latin1', 3000)},
-          {'description': gen_string('html', 3000)})
-    def test_negative_update_3(self, test_data):
```

Comment:
Actually, the ``--description`` flag is not even present in the compose (see [BZ 1192329](https://bugzilla.redhat.com/show_bug.cgi?id=1192329) )